apps/server/src/modules/user-login-migration/service/school-migration.service.spec.ts

```diff
@@ -250,4 +250,48 @@ describe('SchoolMigrationService', () => {
 			);
 		});
 	});
+
+	describe('restartMigration is called', () => {
+		it('should call getSchoolById on schoolService', async () => {
+			const expectedSchoolId = 'expectedSchoolId';
+			const users: Page<UserDO> = new Page([userDoFactory.buildWithId()], 1);
```

Comment:
Page is imported from interface folder and has a constructor. This sound special. The Page should be moved to a other place. I also not sure what is mean with page? 
Should it be Promise<Counted<XXX[]>> ?

export type Counted<T> = [T, number];

The mapping to pagianted response should be done in controller layer with 
PaginationResponse<UserDO[]>. Not in repo or domain layer.

Reply:
Yes, that is correct, i moved it to domain. But dont know if it is the right place.
The term page is a common term. Page is a wrapper object for an array of do's. which is used from services for pagination.

Yes, if we would need the result then we should map in a PaginationResponse in the controller layer.